diff --git a/node_modules/react-native-modal-datetime-picker/src/DateTimePickerModal.android.js b/node_modules/react-native-modal-datetime-picker/src/DateTimePickerModal.android.js
index 3495d1b..bc38900 100644
--- a/node_modules/react-native-modal-datetime-picker/src/DateTimePickerModal.android.js
+++ b/node_modules/react-native-modal-datetime-picker/src/DateTimePickerModal.android.js
@@ -6,7 +6,9 @@ import DateTimePicker from "@react-native-community/datetimepicker";
 const areEqual = (prevProps, nextProps) => {
   return (
     prevProps.isVisible === nextProps.isVisible &&
-    prevProps.date.getTime() === nextProps.date.getTime()
+    // Add Optional Chaining as a workaround for 
+    // issue https://difisoft.sentry.io/issues/4650279935/events/ee26ff7813ed496895f12952af48c777/
+    prevProps.date?.getTime?.() === nextProps.date?.getTime?.()
   );
 };
 
